src/collections/members/_member-profile-template/index.mdx

```diff
@@ -1,15 +1,25 @@
 ---
-name: person // First Last Name
-position: // (e.g. Contributor, Maintainer, MeshMate, Community Manager)
-image_path: ./display-picture.png
-github: // just the user handle (e.g. vinayaksh42)
-twitter: // just the user handle (e.g. Vinayak47427793)
-linkedin: // last portion of https://www.linkedin.com/in/<profile-name> (e.g. vinayak-sharma-141096193)
-location: // City, Country
-bio: // One or two paragraphs about the community member.
-badges:
-  - // (e.g. meshery, community member etc.)
-blurb:
-status: // Active or Inactive
-published: false
+name:       // First and Last Name
+position:   // (e.g. Contributor, Maintainer, MeshMate, Community Manager)
+image_path: ./display-picture.jpg
+github:     // just the user handle (e.g. vinayaksh42)
+twitter:    // just the user handle (e.g. Vinayak47427793)
+linkedin:   // last portion of https://www.linkedin.com/in/<profile-name> (e.g. vinayak-sharma-141096193)
+location:   // City, Country
+bio:        // One or two paragraphs about the community member, enclosed in quotation marks.
+badges:     // List of badges that this member carries. 
+            // Badges represent projects that a member has contributed to.
+            // Possible badges include:
```

Comment:
@leecalcote did you check the build of the site as well?......I doubt that this might cause trouble with that🤔

Reply:
I am sorry, but I thought the template expected to be used is same as other members profiles?

Reply:
@Nikhil-Ladha, no, I didn't confirm that the site is building with these changes. If it isn't building, I might have a commented line out of place. I'll checkout and build to confirm.

Reply:
@Anita-ihuman this updated template accounts for all possible attributes of a community member profile and offers a generic template (a reference) to use when adding new or updating existing profiles.

Reply:
Alright then, this template looks more detailed than the previous one tho.

Reply:
@Anita-ihuman spot on. Hence, the need for this update.

Reply:
@Nikhil-Ladha, I just checked out this branch locally and built. No issues there. Perhaps, a failed DCO check (because of the update via GitHub web) is causing confusion here.

Reply:
Cool.... actually there was some issues earlier with commenting in mdx...so thought of confirming it 😅...as we don't have the checks running currently.

Reply:
@Anita-ihuman thank you for reviewing this. This update will help and make sense in your other PR on Neil Alvares.